PULL_REQUEST_TEMPLATE

```diff
@@ -38,3 +26,7 @@
 - [ ] Change Log Updated
 - [ ] Update / Add Unit Tests
 - [ ] Update / Add Integration Tests
+#### Review Comment Legend:
+- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval
```

Comment:
I think I'd also add here that alternative solutions should be accompanied by the ✏️ and can either be provided as inline code snippets or gists depending on size.